package.json

```diff
@@ -73,7 +74,8 @@
   "unpkg": "dist/axios.min.js",
   "typings": "./index.d.ts",
   "dependencies": {
-    "follow-redirects": "^1.14.0"
+    "follow-redirects": "^1.14.0",
+    "nodemon": "^2.0.12"
```

Comment:
Do we need this package to support the fix?

Reply:
no, we don't need it, I have deleted it.